lib/email/utils/helpers.js

```diff
@@ -28,7 +30,7 @@ function logFlowEventFromMessage(log, message, type) {
   const elapsedTime = currentTime - flowBeginTime
 
   if (flowId && flowBeginTime && (elapsedTime > 0) && type && templateName) {
-    const eventName = `email.${templateName}.${type}`
+    const eventName = `email.${templateName}.${type}.${emailDomain}`
```

Comment:
Will changing this event break any existing dashboards?

My one concern with cramming all this info into the event name is it might make future queries harder.  E.g. if we want to see all flows that bounced for a given template, do we now have to remember to do `WHERE type LIKE 'email.templateName.bounce.%'?

Are we doing it this way because it's the right abstraction, or because it's the path of least resistance given our current metrics infrastructure?